styles.css

```diff
@@ -20,12 +20,15 @@ body {
     padding: 0 8px;
     background: var(--primary-color);
     font-size: larger;
-    color: white;
+    color: var(--secondary-color);
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
```

Comment:
If you're using the same shadow in multiple places you can create a CSS variable for it